ARC specification.md

```diff
@@ -296,102 +302,156 @@ Notes:
+Nevertheless, a mechanism to continuously report the current state and quality of an ARC is indispensable. 
 
-ARCs MAY be validated against 0 or more targets defined in a [validation_targets.yml file](#structure-of-the-validation_targetsyml-file), where the following criteria MUST be met for each target:
-- the target MUST have a unique _name_ across all validation targets used the ARC. This name MUST used for specifying the target in the [validation_targets.yml file](#structure-of-the-validation_targetsyml-file) and the subfolder names in the [validation branch](#structure-of-the-validation-branch)
-- target MUST create a `validation_report.*` file that summarizes the results of validating the ARC against the cases defined in the target. The format of this file SHOULD be of an established test result format such as [JUnit XML](https://github.com/windyroad/JUnit-Schema) or [TAP](https://testanything.org/).
-- the target MUST create a `badge.svg` file that visually summarizes the results of validating the ARC against the cases defined in the target. The information displayed SHOULD be derivable from the `validation_report.*` file and MUST include the _name_ of the target.
+## Validation
 
-### Structure of the validation branch
+The process of assessing quality parameters of an ARC is further referred to as _validation_ of the ARC against a [_validation package_](#validation-packages), where the _validation package_ is an arbitrary set of [validation cases](#validation-cases) that the ARC MUST pass to qualify as _valid_ in regard to the _validation package_.
 
-To make sure that validation results are bundled with the ARC but do not pollute the commit history, validation results MUST be stored in a separate branch of the ARC repository. This branch:
-- MUST be named `validation`
+### Validation cases
+
+A **validation case** is the atomic unit of a [validation package](#validation-packages) describing a single, deterministic and reproducible requirement that the ARC MUST satisfy in order to qualify as _valid_ in regard to it.
+
+Format and scope of these cases naturally vary depending on the type of ARC, aim of the containing validation package and tools used for creating and performing the validation. 
+Therefore, no further requirements are made on the format of validation cases.
+
+  example:
+
+  > The following example shows a validation case simply defined using natural language.
+
+  ```
+  All Sample names in this ARC must be prefixed with the string "Sample_"
+  ```
+
+  Any ARC where all sample names are prefixed with the string "Sample_" would be considered valid in regard to this validation case.
+
+### Validation packages
+
+A **validation package** bundles a collection of [validation cases](#validation-cases) that the ARC MUST pass to qualify as _valid_ in regard to the _validation package_ with instructions on how to perform the validation and summarize the results.
+
+Validation packages
+
+- MUST be executable. 
```

Comment:
So validation cases don't need to be executable (see "no further requirements are made on the format of validation cases") but the packages do?

Reply:
yes